templates/registry/registry-svc.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 {{ include "harbor.labels" . | indent 4 }}
 spec:
   ports:
-    - name: {{ ternary "https-registry" "http-registry" .Values.internalTLS.enabled }}
+    - name: registry
```

Comment:
This is the `http-` prefix which I had to drop to make it work in our istio setup.

Although I'd like to see this getting merged I think it would be better to introduce an option to override the service name, something like

```
registry:
  service:
    name: registry
```

that would take precedence if defined